app/controllers/v0/claims_controller.rb

```diff
@@ -4,13 +4,18 @@ class ClaimsController < ApplicationController
     skip_before_action :authenticate
 
     def index
-      render json: current_user.claims,
+      render json: Claim.fetch_all(current_user.vaafi_headers),
```

Comment:
So take a look at what we're currently doing with APIs, i was thinking those could get moved into app/resources or something and maybe, even get mixed in to models with our own custom finder methods. But thats all open to feedback and we can hopefully all arrive on a reasonable consensus.